visualizations/fan_chart/Makefile

```diff
@@ -11,7 +11,7 @@ install: build
 	pip install $(ARGS) .
 
 test:
-	py.test --cov=webviz_fan_chart --cov-fail-under=100 tests/
+	py.test --cov=webviz_fan_chart --cov-report=term-missing --cov-fail-under=100 tests/
```

Comment:
This is ok, but we do not need to make this change. Better to leave it out for a cleaner commit.

Reply:
Should we maybe make this a default option? I tend to use it all the time to see why my test rate is not 100%, and since it is not making any difficulties or issues it could be a nice addition.

Reply:
Maybe you could add a separate commit to this PR that changes it for all Makefiles?